libs/tcp_wrappers/patches/006-compilation-warnings.patch

```diff
@@ -1,12 +1,12 @@
 diff -u tcp_wrappers_7.6.orig/clean_exit.c tcp_wrappers_7.6/clean_exit.c
 --- tcp_wrappers_7.6.orig/clean_exit.c	1994-12-29 03:42:20.000000000 +1100
-+++ tcp_wrappers_7.6/clean_exit.c	2017-11-14 22:17:24.000000000 +1100
++++ tcp_wrappers_7.6/clean_exit.c	2017-11-14 22:50:48.000000000 +1100
 @@ -9,10 +9,11 @@
    */
  
  #ifndef lint
 -static char sccsid[] = "@(#) clean_exit.c 1.4 94/12/28 17:42:19";
-+static char __attribute__((__unused__)) sccsid[] = "@(#) clean_exit.c 1.4 94/12/28 17:42:19";
++static char sccsid[] __attribute__((__unused__)) = "@(#) clean_exit.c 1.4 94/12/28 17:42:19";
```

Comment:
You unfortunately can't also use `__attribute_unused__` from `<cdefs.h>` because header files don't get included until much later, but you could use `-D_Unused="__attribute__((__unused__)"` in `CFLAGS` and then short-hand it as `_Unused`.

In any case, please rebase+squash.